runtime-sdk/src/crypto/signature/sr25519.rs

```diff
@@ -60,6 +61,27 @@ impl PublicKey {
             .map_err(|_| Error::VerificationFailed)
     }
 
+    /// Verify a signature.
+    pub fn verify_raw(
```

Comment:
This is not in line with other signers, where you have ` /// Verify signature without using any domain separation scheme.`, so optionally  we could think about renaming this method.

Reply:
This matches the polkadot implementation.